module/VuFindTheme/src/VuFindTheme/ThemeGenerator.php

```diff
@@ -46,6 +47,25 @@ class ThemeGenerator extends AbstractThemeUtility implements GeneratorInterface
 {
     use \VuFindConsole\ConsoleOutputTrait;
 
+    /**
+     * Config file path resolver
+     *
+     * @var PathResolver
+     */
+    protected $pathResolver;
+
+    /**
+     * Constructor
+     *
+     * @param ThemeInfo    $info         Theme info object
+     * @param PathResolver $pathResolver Config file path resolver
+     */
+    public function __construct(ThemeInfo $info, PathResolver $pathResolver = null)
```

Comment:
We need to adjust a factory to inject this, don't we?

Reply:
Oops, added.